### PR TITLE
fix new doc for diskful new kernel patch

### PR DESCRIPTION
**Pull request by bybai**
issue: #3348 

test link: http://10.4.41.1/install/kernelupdate/xcat-core/docs/build/html/guides/admin-guides/manage_clusters/ppc64le/diskful/customize_image/use_kernel_patch.html

### CODE REVIEW

**Comment by immarvin** [comment]
hi @bybai , the steps looks good.  Since you have also tried the kernel update for diskless, I think there should also be a section for diskless

**Comment by bybai** [comment]
Hi @immarvin , We have one page in readthedocs for install new kernel for diskless: http://xcat-docs.readthedocs.io/en/latest/guides/admin-guides/manage_clusters/ppc64le/diskless/customize_image/install_new_kernel.html

File: docs/source/guides/admin-guides/manage_clusters/common/deployment/use_kernel_patch.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+The RPM names below are only examples, substitute your specific level and architecture.
+
+* **[RHEL]**
+
+#. The RPM kernel package is usually named: kernel-<kernelver>.rpm. Append new kernel packages directory to osimage pkgdir ::
+
+        mkdir -p /install/kernels/<kernelver>
+        cp /tmp/kernel-*.rpm /install/kernels/<kernelver>
+        createrepo /install/kernels/<kernelver>/
+        chdef -t osimage rhels7.3-ppc64le-install-compute pkgdir=/install/rhels7.3/ppc64le,/install/kernels/<kernelver>
```
Comment (immarvin) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/use_kernel_patch.rst]: should better be ``chdef -t osimage rhels7.3-ppc64le-install-compute -p pkgdir=/install/kernels/<kernelver>``

**Comment by immarvin** [comment]
agree to merge

**Comment by daniceexi** [comment]
@bybai I think this page should be work together with page http://xcat-docs.readthedocs.io/en/latest/guides/admin-guides/manage_clusters/ppc64le/diskful/customize_image/driver_update_disk.html?highlight=inject

**Pull request merged**